app/models/custom/poll/question.rb

```diff
@@ -0,0 +1,10 @@
+require_dependency Rails.root.join("app", "models", "poll", "question").to_s
+
+class Poll::Question < ApplicationRecord
+  translates :description, touch: true
+  globalize_accessors
+
+  def is_single_choice?
```

Comment:
I think we usually don't add `is_` prefixes since the question mark makes the verb implicit:

```suggestion
  def single_choice?
```